Resources/Textures/SS220/Objects/Consumable/Drinks/glass_berry.rsi/meta.json

```
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Added by gabova(discord)",
```

```suggestion
    "copyright": "Taken from https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi and modified by gabova(discord)",
```
Оригинальный спрайт стакана, как и напитков, сделан другим человеком и залит под CC-BY-SA-3.0
Его модификации должны быть подписаны, как модификации, по условию лицензии.